Controller/CRUDController.php

```diff
@@ -221,7 +221,8 @@ public function deleteAction($id)
     /**
      * Edit action.
      *
-     * @param int|string|null $id Optional ID to use instead of globally provided default.
+     * @param int|string|null $id Optional ID representing the main admin subject.
```

Comment:
> IMO, this parameter should be deprecated and removed, unless we find a very good reason for it. Maybe you have one @curry684 ?

In another way, I think we should instead reintroduce request parameter directly on action (And maybe param fetcher) instead of playing each time with the request.

I think we can use this PR for now on work for this.